src/debugger/Stepper.js

```diff
@@ -57,3 +58,42 @@ export class StepIntoStepper extends Stepper {
     return true;
   }
 }
+
+export class StepOverStepper extends Stepper {
+  constructor(filePath: string, line: number, column: number, stackSize: number) {
+    super(filePath, line, column);
+    this._startStackSize = stackSize;
+  }
+  _startStackSize: number;
+
+  isComplete(ast: BabelNode, realm: Realm): boolean {
+    // we should only step to statements
+    if (!IsStatement(ast)) return false;
+    let loc = ast.loc;
```

Comment:
I have seen this block of code again and again for at least three times. Take them out into a helper function.